client/checkout/utils/upe.js

```diff
@@ -36,3 +36,31 @@ export const isWCPayChosen = function () {
 	return document.getElementById( 'payment_method_woocommerce_payments' )
 		.checked;
 };
+
+/**
+ * Returns the cached payment intent for the current cart state.
+ *
+ * @param {Object} paymentMethodsConfig Array of configs for payment methods.
+ * @param {string} paymentMethodType Type of the payment method.
+ * @return {Object} The intent id and client secret required for mounting the UPE element.
+ */
+export const getPaymentIntentFromSession = (
+	paymentMethodsConfig,
+	paymentMethodType
```

Comment:
Since you've just updated this function signature, can you please pass the `paymentMethodsConfig` parameter through when this function is called in the classic checkout over [here](https://github.com/Automattic/woocommerce-payments/blob/fix/4603-block-split-upe/client/checkout/classic/upe.js#L706)? I think this currently breaks the classic checkout, but should be an easy fix!

Reply:
Oh, good catch! Thanks for noticing and sorry for not checking the classic checkout flow after all the changes. [It's fixed now](https://github.com/Automattic/woocommerce-payments/pull/4681/commits/c16cfec392a41b1328280bb1b6d8159c323c22c2) and should work as expected.